lib/block_view_derived_key_test.go

```diff
@@ -508,7 +667,7 @@ func _getAuthorizeDerivedKeyMetadataWithTransactionSpendingLimitAndDerivedPrivat
 	expirationBlockByte := EncodeUint64(expirationBlock)
 	accessBytes := append(derivedPublicKey, expirationBlockByte[:]...)
 
-	transactionSpendingLimitBytes, err := transactionSpendingLimit.ToBytes()
+	transactionSpendingLimitBytes, err := transactionSpendingLimit.ToBytes(0)
```

Comment:
hmm we should probably be able to test before and after the block height. we can modify the block height at which the fork occurs by updating params.ForkHeights, so we may need to pass a height in here?

Reply:
I did a very hardcore before - at - after block height test, lmk what you think